index.js

```diff
@@ -112,48 +112,52 @@ function fetchDataByView(done, st = state.Home) {
         });
       break;
     case "Blog":
```

Comment:
The 'Blog' code has also mostly been gutted out of the SPA so this Blog code should go as well right?